internal/transform/schema.go

```diff
@@ -102,7 +102,7 @@ type OperationOutput struct {
 	SourceAccount      string                 `json:"source_account"`
 	SourceAccountMuxed string                 `json:"source_account_muxed,omitempty"`
 	Type               int32                  `json:"type"`
-	ApplicationOrder   int32                  `json:"application_order"`
+	TypeString         string                 `json:"type_string"`
```

Comment:
it's a bit wierd to see type and typestring within the same struct. Maybe in the future we could consider using type and subtype ?